packages/pages-shared/__tests__/asset-server/handler.test.ts

```diff
@@ -1069,6 +1069,168 @@ describe("asset-server handler", () => {
 			);
 		});
 	});
+
+	test("should emit header when Web Analytics Token is injected", async () => {
+		const findAssetEntryForPath = async (path: string) => {
+			if (path === "/index.html") {
+				return "asset-key-index.html";
+			}
+			return null;
+		};
+
+		const fetchAsset = async () =>
+			Promise.resolve(
```

Comment:
Since the function is `async` I don't think you need this `Promise.resolve` here.

Reply:
Alternatively, remove the `async`.